### DISCUSSION TITLE
2023-07-26

### DISCUSSION THREAD

**Discussion by woosung1223** [woowacourse-teams/2023-haru-study#137] [daily_meeting]
### 오늘의 할 일
- 서비스 론칭 완료 및 피드백 반영사항 기능 구현 (오후 3시 반까지)
    - 프론트, 백 미구현 기능 구현 및 리팩토링 진행
    - 프론트, 백 서비스 연동 완료
    - 배포 DB 드랍방지 설정
    - 정식 론칭 서비스 배포 완료
    - 시간 선택 범위 확대하기
- 소켓, 폴링 양방향 통신 어떻게 할지
### 언젠가 해야할 일
- 기획 변경된 이유 정리
- DNS 어떻게 할지
- 질문 한 두개만 빼고 선택으로 바꾸기
- 질문 PlaceHolder로 유의미한 응답 예시를 제공하는 기능 (목표설정 가이드라인)
- 브랜치 전략 결정하기(dev -> main)

**Comment by yeopto** [comment]
### 체크인

7점 - 감기기운이 있어서 -3점입니다. 잠은 푹 잤네요 그래도

**Comment by nlom0218** [comment]
### 체크인

10점 - 오늘 배포할 수 있다니! 설렙니다 😁

**Comment by jaehee329** [comment]
### 체크인
7점 - 그럭저럭...
### 체크아웃
7점 - 저럭그럭

**Comment by MoonJeWoong** [comment]
### 체크인
9점 - 오늘 아침에 크론이랑 얘기해서 재미있었습니다! 그런데 원래 하려고 했던 공부는 적게 해서 1점 깎았습니다 ㅎㅎ

**Comment by aak2075** [comment]
### 체크인

10점 - 점심 먹을 생각에 신나요

**Comment by woosung1223** [comment]
### 체크인
8점 - 기분이 나쁘지는 않은데 어제 늦게 자서 -2점!!

**Comment by woo-jk** [comment]
### 체크인
10점 - 점수 깎일 요소가 없다~!
